src/components/ToDoHeader.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div class="flex justify-between items-center font-header t-0 w-full">
+    <div class="flex justify-center items-center ">
+      <p class="w-72 h-20 left-105 top-20.5 non-italic font-semibold text-3xl md:text-5xl lg:text-7xl text-black tracking-tighter text-left mt-3">To do list</p>
+    </div>
+    <div @click="emit('show')" class="h-16 w-16 mb-8 rounded-full outline-none border-none left-72 bg-transparent flex justify-center items-center hover:scale-110 transition-transform ease-in-out duration-300 cursor-pointer">
+        <svg width="61" height="60" viewBox="0 0 61 60" fill="none" xmlns="http://www.w3.org/2000/svg" class="w-full h-full">
```

Comment:
Create an icon component from this, ex PlusIcon.vue inside components/icons folder